docs/eigenda/overview.md

```diff
@@ -5,8 +5,7 @@ title: EigenDA Overview
 
 EigenDA is a data availability store made by
```

Comment:
Reword this entire section as:

EigenDA, developed by EigenLabs, is a data availability solution deployed on EigenLayer, and is operational on Ethereum mainnet since April 2024. For development and testing endeavors, it is also accessible on the Holesky testnet.